### PR TITLE
docs: Enable Javadoc generation as GitHub action

### PR DESCRIPTION
**Pull request by o-kopysov**
# Description

Here I want to enable Javadoc generation and publication on Github pages.
In this PR I covered `com.lpvs` and `com.lpvs.controller` packages by comments. 
Other packages will be covered by Javadoc comments step-by-step in other PRs.
Also, I've created the task #307 with the request to update comments.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [X] Documentation update
- [X] This change requires a documentation update
- [X] CI system update
- [ ] Test Coverage update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tdrozdovsky — APPROVED** [review]

**Review by userai-dev — APPROVED** [review]
Good job, approved.

File: src/test/java/com/lpvs/controller/GitHubWebhooksControllerTest.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public void wrongSecretTest() {
                         + "}"
                         + "}";
         try {
-            gitHubWebhooksController.setProps();
+            gitHubWebhooksController.initializeGitHubSecret();
```
Comment (v-kerimov) [inline src/test/java/com/lpvs/controller/GitHubWebhooksControllerTest.java:151]: @o-kopysov Does this also relates to the Javadoc feature, or was it just some fix or refactoring that came to your attention?

File: src/test/java/com/lpvs/controller/LPVSWebControllerTest.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public void testPersonalInfoSettings() {
         when(loginCheckService.getMemberFromMemberMap(authentication)).thenReturn(member);
 
         LPVSMember result =
-                webController.new PublicInterface().personalInfoSettings(authentication);
+                webController.new WebApiEndpoints().personalInfoSettings(authentication);
```
Comment (v-kerimov) [inline src/test/java/com/lpvs/controller/LPVSWebControllerTest.java:69]: Same question with this test file.

**Review by v-kerimov — APPROVED** [review]

File: src/test/java/com/lpvs/controller/GitHubWebhooksControllerTest.java (hunk 1 above)
Comment (o-kopysov) [reply src/test/java/com/lpvs/controller/GitHubWebhooksControllerTest.java:151]: While preparing Javadocs, I did some name refactoring to make function names more understandable.

File: src/test/java/com/lpvs/controller/LPVSWebControllerTest.java (hunk 2 above)
Comment (o-kopysov) [reply src/test/java/com/lpvs/controller/LPVSWebControllerTest.java:69]: After updating the name of an interface, I have to update UTs as well.

File: src/main/java/com/lpvs/controller/package-info.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/**
```
Comment (m-rudyk) [inline src/main/java/com/lpvs/controller/package-info.java:1]: Still, copyright header required here

File: src/main/java/com/lpvs/package-info.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/**
```
Comment (m-rudyk) [inline src/main/java/com/lpvs/package-info.java:1]: same as above

**Review by m-rudyk — APPROVED** [review]
Approved.

File: src/main/java/com/lpvs/controller/package-info.java (hunk 3 above)
Comment (o-kopysov) [reply src/main/java/com/lpvs/controller/package-info.java:1]: Thank you!

**Pull request merged**